views-fieldset/src/main/java/io/micronaut/views/fields/InputType.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2017-2023 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.views.fields;
+
+import io.micronaut.core.annotation.Experimental;
+
+/**
+ * Constants for the value of the type attribute in an HTML input tag.
+ * @author Sergio del Amo
+ * @since 4.1.0
+ */
+@Experimental
+public final class InputType {
```

Comment:
Why not an interface or an enum?

Reply:
resolved via https://github.com/micronaut-projects/micronaut-views/pull/613/commits/77dc8ee6c32ad5eca44bed2b63d6e8bfd8ba28ff